nowcasting_dataset/data_sources/nwp/nwp_data_source.py

```diff
@@ -181,6 +181,16 @@ def open_nwp(zarr_path: str, consolidated: bool) -> xr.DataArray:
 
     ukv = nwp["UKV"]
```

Comment:
is the reversing needed for other channels?

Reply:
I may have miss understood here - UKV might not be a channel. Perhaps add a comment why "UKV" is selected

Reply:
Ah, yes, good point!  `UKV` is the name of the (one and only) `DataArray` in the Zarr `Dataset`.  You're right, `UKV` isn't a channel.  `UKV` is the name of the DataArray which holds all the channels (the channels are represented as a dimension in the DataArray).  I'll add a comment!  Thanks!